tests/cephadm/test_verify_crash_reports.py

```diff
@@ -0,0 +1,43 @@
+from cli.cephadm.cephadm import CephAdm
+from cli.exceptions import OperationFailedError
+from cli.utilities.utils import get_all_running_pids, kill_process
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """Verify redeploy for a specific service
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
+    """
+    # Check for mgr nodes
+    installer = ceph_cluster.get_nodes(role="installer")[0]
+    fsid = CephAdm(installer).ceph.fsid()
+
+    # Identify the osd pid
+    out = get_all_running_pids(installer, "osd")
+    if not out:
+        raise OperationFailedError("No osd daemons running")
+
+    # Introduce a crash in OSD daemon
+    kill_process(installer, out)
+
+    # Get crash stats
+    crash_stat = CephAdm(installer).ceph.crash.stat()
+    log.info(f"Crash stats after stopping container are '{crash_stat}'")
+
+    # Validate if the crash is posted in /var/lib/ceph/<fsid>/crash, /var/lib/ceph/<fsid>/crash/posted directories.
+    folders = [f"/var/lib/ceph/{fsid}/crash", f"/var/lib/ceph/{fsid}/crash/posted"]
+    log_found = False
+    for folder in folders:
+        dirs = installer.get_dir_list(dir_path=folder, sudo=True)
+        if dirs:
+            log_found = True
+    if not log_found:
```

Comment:
no need to have this logic, test case should fail even one directory fails to generate crash logs.